src/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import markdownList from './reducerMarkdownList';
+import document from './reducerDocument';
+
+export default combineReducers({
+  markdownList,
+  document
```

Comment:
consider changing this prop. `document` already exists in browsers. Maybe call it `doc`?

https://trello.com/c/3dHxG1VF/12-use-rename-document-in-combinedreducer